### PR TITLE
updated intent diagram

### PR DESCRIPTION
**Pull request by KayleeLChan**
## Pull Request Summary

Fixed the formatting on intent diagram! Please QA and let me know if it works nicely on your screen aspect ratio!

### Change types

- [x] Front end change

### CODE REVIEW

File: front-end/src/components/analysis-page/IntentDiagram.jsx [hunk 1]
```diff
@@ -98,7 +98,7 @@ class IntentDiagram extends Component {
                         </label>
                     </button>
                 </div>
-                <div className="rounded-lg flex flex-col gap-y-3 md:flex-row mx-auto justify-between text-center gap-x-9 m-9">
+                <div className="rounded-lg flex flex-col gap-y-3 md:flex-row mx-auto justify-evenly text-center overflow-auto gap-x-5 m-9 sm:min-w-[37rem] sm:max-w-[35rem] 2xl:max-w-[50rem] 2xl:max-h-[14rem]">
```
Comment (WeissSa) [inline front-end/src/components/analysis-page/IntentDiagram.jsx:101]:
```suggestion
                <div className="rounded-lg flex flex-col gap-y-3 md:flex-row mx-auto md:justify-evenly text-center md:overflow-auto gap-x-5 m-9 sm:min-w-[37rem] sm:max-w-[35rem] 2xl:max-w-[50rem] 2xl:max-h-[14rem]">
```
or else we get
image

**Review by WeissSa — CHANGES_REQUESTED** [review]
Great other than this one issue. I think my suggestion will fix it :)

**Review by emilylwan — APPROVED** [review]
Love the small changes!

**Review by helias22 — CHANGES_REQUESTED** [review]
Screen Shot 2022-12-01 at 12 31 43 AM

I don't think the first child is located correctly 🤔

**Comment by WeissSa** [comment]
Anyone know why I am not able to analyze a transcript on the preview build for this?

**Comment by WeissSa** [comment]
also after testing on local, removing justify-evenly fixes helia's issue @KayleeLChan

**Review by WeissSa — APPROVED** [review]
yassified

**Pull request merged**